spec/services/step_by_step_page_reverter_spec.rb

```diff
@@ -68,6 +68,10 @@
       expect(step_by_step.draft_updated_at).to eq(published_at)
     end
 
+    it "sets the status to published" do
+      expect(step_by_step_page.status).to eq("published")
```

Comment:
If `StringInquirer` is used for status, then I think this spec could use a predicate matcher

```ruby
expect(step_by_step_page.status).to be_published
```

Reply:
Update and fixed-up